src/main/java/core/basesyntax/SalaryInfo.java

```diff
@@ -1,7 +1,48 @@
 package core.basesyntax;
 
+import java.text.ParseException;
+import java.text.SimpleDateFormat;
+import java.util.Date;
+
 public class SalaryInfo {
     public String getSalaryInfo(String[] names, String[] data, String dateFrom, String dateTo) {
-        return null;
+        Date dateFromDate = new Date();
```

Comment:
[Use LocalDate instead of Date.](https://mate-academy.github.io/jv-program-common-mistakes/java-core/exceptions/salary-info#use-localdate-instead-of-date)